### ISSUE TITLE
Scroll bars are almost invisible

### ISSUE THREAD

**Issue by walles** [nickcernis/blackbird-jetbrains#2]
Hi!

Trying this on IntelliJ 2021.2 Community Edition / macOS Catalina 10.15.7.

macOS color theme is the dark one.

As can be seen in the screenshot, the right-hand side scrollbar is almost invisible.

Since the scroll bars auto hide when not used (I think this is an OS thing), maybe make the scrollbar quite bright?

<img width="362" alt="Skärmavbild 2021-09-03 kl  13 33 45" src="https://user-images.githubusercontent.com/158201/131998907-d3b511aa-5dc3-4e1e-9ceb-1fb87ec27075.png">

**Comment by nickcernis** [comment]
Thanks for the report, Johan!

I'll look into increasing the contrast or using a bright colour when scrollbars are moving.

**Comment by nickcernis** [comment]
I submitted a new version with a fix (0.3.3). It's awaiting manual review but should be out next week.

I tried a bright colour but it was pretty distracting when scrolling, so I went with a higher contrast when the scrollbar is moving instead. Thanks again for the feedback.

<img width="236" alt="Screenshot 2021-09-03 at 21 44 05" src="https://user-images.githubusercontent.com/647669/132059311-6a0519c7-ee21-431c-bde1-abfcbb53f1e6.png">
